src/entities/CollectionToken/CollectionTokenTransformers.ts

```diff
@@ -24,16 +24,20 @@ const transformErc1155TokenAttributeToCollectionTokenAttribute = (attribute: Erc
   value: `${attribute.value}`,
 });
 
-export const transformErc1155TokenToCollectionToken = (token: Erc1155Token, tokenId: number, price: number): CollectionToken => ({
-  id: tokenId,
-  image: (token.extensions.metadata.image_url || token.extensions.metadata.image).replace('ipfs://', 'https://ipfs.io/ipfs/'),
-  description: token.extensions.metadata.description,
-  price,
-  name: token.extensions.metadata.name,
-  attributes: (token.extensions.metadata.attributes || []).map(transformErc1155TokenAttributeToCollectionTokenAttribute),
-  createdBy: token.extensions.metadata.created_by,
-  externalUrl: token.extensions.metadata.external_url,
-});
+export const transformErc1155TokenToCollectionToken = (token: Erc1155Token, tokenId: number, price: number): CollectionToken => {
+  console.log(token);
```

Comment:
Don' think that needs to be there :stuck_out_tongue:
(But I can't seem to suggest the code :man_shrugging: )